CONTRIBUTING.md

```diff
@@ -173,9 +173,27 @@ both high-quality and timely.
 
 Finally, if nothing brings you more satisfaction than seeing every last issue
 labeled and all resolved issues closed, feel free to message any Eyre Circle
-Member (currently @yaahc) for the triage role to help us keep things tidy. This
+Member (currently @yaahc and @ten3roberts) for the triage role to help us keep things tidy. This
 role only requires good faith and a basic understanding of our development
 process.
 
 [Discord]: https://discord.gg/z94RqmUTKB
 [^1]: Okay, I'll admit it, it's really just the Rust Project's CoC :sweat_smile:
+
+# Release Process
```

Comment:
We can always refine this, but we need something out now and ensure that releases go smoothly and don't drop commits